tgui/packages/tgui/interfaces/CrewCredits.tsx

```diff
@@ -0,0 +1,79 @@
+import { BooleanLike } from "common/react";
+import { useBackend } from "../backend";
+import { Icon, Section, Stack } from "../components";
+import { Window } from "../layouts";
+
+interface CrewCreditsData {
+  groups: GroupBlockProps[]
+}
+
+export const CrewCredits = (props, context) => {
+  const { data } = useBackend<CrewCreditsData>(context);
+  return (
+    <Window title="Crew Credits" width={600} height={600}>
+      <Window.Content scrollable>
+        {data.groups?.map((group, index) =>
+          ((data.groups[index].crew.length > 0) && <GroupBlock key={index} group={group.group} crew={group.crew} />)
+        )}
+      </Window.Content>
+    </Window>);
+};
+
+interface GroupBlockProps {
+  group: string,
+  crew: CrewMemberProps[]
+}
+
+const GroupBlock = (props:GroupBlockProps) => {
+  const { group: group_title, crew } = props;
+  const dep_heads = crew?.filter(member => member.head);
+
+  return (
+    <Section title={group_title}>
+      <Stack fill vertical>
+        {dep_heads?.map((member, index) =>
+          (
+            <CrewMember
+              key={"head" + index}
+              real_name={member.real_name}
+              dead={member.dead}
+              player={member.player}
+              role={member.role}
+              head />
+          )
+        )}
+        {crew?.map((member, index) =>
+          (
+            !member.head && <CrewMember
+              key={index}
+              real_name={member.real_name}
+              dead={member.dead}
+              player={member.player}
+              role={member.role} />
+          )
+        )}
```

Comment:
i'll be honest with you, we kinda need to have 2 maps in case some department inexplicably has multiple heads